openslides_backend/main.py

```diff
@@ -50,7 +50,7 @@ def load_config(self) -> None:
         options = {
             "bind": DEFAULT_ADDRESSES[self.view_name],
             "worker_tmp_dir": "/dev/shm",  # See https://pythonspeed.com/articles/gunicorn-in-docker/
-            "timeout": int(os.environ.get("OPENSLIDES_BACKEND_WORKER_TIMEOUT", "30")),
+            "timeout": int(os.environ.get("OPENSLIDES_BACKEND_WORKER_TIMEOUT", "0")),
```

Comment:
Please check this default value again. What timeout do we want in production? See https://docs.gunicorn.org/en/stable/settings.html#timeout

Reply:
see https://github.com/OpenSlides/openslides-backend/pull/1028#issuecomment-972678259 and the email from @peb-adr: this is a temporary fix.